Doc/howto/clinic.rst

```diff
@@ -93,67 +96,143 @@ and it should be able to do many interesting and smart
 things with all the information you give it.
 
 
-Basic concepts and usage
-------------------------
+Basic concepts
+--------------
 
-Argument Clinic ships with CPython; you'll find it in ``Tools/clinic/clinic.py``.
-If you run that script, specifying a C file as an argument:
+When Argument Clinic is run on a file, either via the :ref:`clinic-cli`
+or via ``make clinic``, it will scan over the input files looking for
+:term:`start lines <start line>`:
 
-.. code-block:: shell-session
-
-    $ python Tools/clinic/clinic.py foo.c
-
-Argument Clinic will scan over the file looking for lines that
-look exactly like this:
-
-.. code-block:: none
+.. code-block:: c
 
     /*[clinic input]
 
-When it finds one, it reads everything up to a line that looks
-exactly like this:
+When it finds one, it reads everything up to the :term:`end line`:
 
-.. code-block:: none
+.. code-block:: c
```

Comment:
This results in wrongly-highlighted output:
![image](https://github.com/python/cpython/assets/25624924/be0cece6-bc14-464a-a002-cd0b98e2b048)

Reply:
That is unfortunate. If there is no workaround, let's revert these particular changes.

Reply:
Reverted for now.